solr/core/src/java/org/apache/solr/cloud/api/collections/CollectionHandlingUtils.java

```diff
@@ -111,7 +111,7 @@
       DocCollection.PER_REPLICA_STATE, null,
       ZkStateReader.PULL_REPLICAS, "0"));
 
-  protected static final Random RANDOM;
+  public static final Random RANDOM;
```

Comment:
My instinct would be to lowercase this field name because it's not really some constant.  It's rather rare in the codebase to use all-caps in other situations.

Reply:
This variable was moved here as part of a refactoring, see [PR 2390](https://github.com/apache/lucene-solr/pull/2390).
When introduced in [2014 in OverseerCollectionMessageHandler](https://github.com/apache/lucene-solr/commit/3f999895448c9524c77ff33b801f43cfb6f8b8bd#diff-942546903f5862bde980b5392f8a4cf99e9b16fe1d48f7d1a0e61661dc36a173R167) this variable already had that uppercase name.

Do you want me to lowercase it? Likely matter of personal taste, but to me it seems ok uppercase.